exonum/src/blockchain/service.rs

```diff
@@ -148,6 +148,17 @@ pub trait Service: Send + Sync + 'static {
     fn state_hash(&self, snapshot: &Snapshot) -> Vec<Hash>;
 
     /// Tries to create a `Transaction` from the given raw message.
+    ///
```

Comment:
Shan't we explicitly say that the _only_ thing that the core guarantees is that `service_id` is equal to the id of this service (and, maybe, the message has proper size), and therefore, the user implementing the interface **must** handle invalid transactions that may come from byzantine nodes?